rules.d/10-systemd-on-move.rules

```diff
@@ -0,0 +1,16 @@
+#  SPDX-License-Identifier: LGPL-2.1+
+#
+#  This file is part of systemd.
+#
+#  systemd is free software; you can redistribute it and/or modify it
+#  under the terms of the GNU Lesser General Public License as published by
+#  the Free Software Foundation; either version 2.1 of the License, or
+#  (at your option) any later version.
+
+ACTION!="move", GOTO="systemd_on_move_end"
+
+# Unset SYSTEMD_ALIAS= property on move event
+SUBSYSTEM=="net", ENV{SYSTEMD_ALIAS}=""
+SUBSYSTEM=="bluetooth", ENV{SYSTEMD_ALIAS}=""
```

Comment:
i don't get this, can you elaborate on this?

Reply:
As I commented at https://github.com/systemd/systemd/issues/16967#issuecomment-688016303, if the device is renamed, then `SYSTEMD_ALIAS=` contains both old and new alias, and the old alias unit will not removed. To remove the old alias, it is necessary to clear the property.

Reply:
hmm, we don#t flush out the env vars on move? that sounds like a bug no?

Reply:
Not sure, but I think we should not. If we change that, the existing rules can be easily broken...
Note that the relevant code is the following: https://github.com/systemd/systemd/blob/12ce0f41735e3d179c41deee99d9abe8aae52527/src/udev/udev-event.c#L947-L951

Reply:
grmbl. it appears wrong to me to copy that over, but this smells indeed like it would break stuff if we'd suddenly change that...

Reply:
So this was introduced in b081b27e1433cdc7ac72b25ae8b4db887d79187f afaics. i.e. it's not like it always was copied. Unfortunately tom gave little explanation in the commit... But it appears it was done to handle network ifaces. Maybe we can just drop this, and announce it via NEWS. Given that we just did the bind/unbind compat breakage, maybe we should do that at the same time.

Of course, we need to make sure that the udev rules the copy replaces are somehow taken care of...